CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Whirlwind](https://github.com/Whirlwind)
   [#7878](https://github.com/CocoaPods/CocoaPods/pull/7878)
 
+* Use CP_HOME_DIR as the base for all default directories  
```

Comment:
nit: I think this is a bug fix, not enhancement. Also can wrap in CP_HOME_DIR in back ticks.

Reply:
I agree, but wasnt sure the maintainers would see it in that way :)